x-pack/legacy/plugins/ml/public/breadcrumbs.js

```diff
@@ -12,3 +12,24 @@ export const ML_BREADCRUMB = Object.freeze({
   }),
   href: '#/'
 });
+
+export const SETTINGS = Object.freeze({
```

Comment:
Not sure where the current breadcrumb for the Data Frame Analytics page is coming from, but I wonder if it should match the text of the menu as in `Data Frame Analytics` rather than just `Analytics`? Or is that over-long?

![image](https://user-images.githubusercontent.com/7405507/64696593-b4804980-d496-11e9-815e-c6188082217b.png)

Reply:
I think having it match should work 👍